api/v1alpha2/linodeobjectstoragekey_types.go

```diff
@@ -90,10 +112,6 @@ type LinodeObjectStorageKeyStatus struct {
 	// +optional
 	LastKeyGeneration *int `json:"lastKeyGeneration,omitempty"`
 
-	// SecretName specifies the name of the Secret containing access key data.
-	// +optional
```

Comment:
Do we no longer need to set the GeneratedSecret.Name in the status after generating a secret for the key? Do we think this might be useful later?

Reply:
Yes, we only want a single source of truth for referring to the secret. We manage the fields in `spec.generatedSecret.name` and `.namespace` so we don't need to replicate those fields in status.